README.md

```diff
@@ -37,7 +37,7 @@ Note the format of the date (`YYYY-MM-DD`) and the fact that multiple chores can
 
 ##### Service account
 
-Create a service account using [these instructions](https://developers.google.com/android/management/service-account). Once it's created, share the previously created spreadsheet with the service account's address. View-only permissions should be fine; we're only reading data from the sheet.
+Create a service account using [these instructions](https://developers.google.com/android/management/service-account). Once it's created, share the previously created spreadsheet with the service account's address. You will need both Read and Write permissions for this to work.
```

Comment:
Please change this line to:

>You will need both `https://www.googleapis.com/auth/drive` and `https://www.googleapis.com/auth/spreadsheets` API access for the bot to work.

Reply:
Do we need `https://www.googleapis.com/auth/drive`? I thought it was just `https://www.googleapis.com/auth/spreadsheets` based on this https://developers.google.com/identity/protocols/googlescopes

Reply:
You would think 😂but you'd be incorrect based on our testing

Reply:
I was unable to get an error by not having `https://www.googleapis.com/auth/drive`. Want to talk about this later on voice? Might be easier to resolve.

Reply:
Just added a commit for this. Could you please check it? Thanks in advance.